modules/stanford_events/lib/scss/stanford_events.node.scss

```diff
@@ -54,11 +54,46 @@
 
   .su-event-dek {
     @include big-paragraph;
-    @include modular-spacing('margin', null null 4 null);
+    @include modular-spacing('margin', null null 2 null);
   }
+}
+
+.node-stanford-event-su-event-sponsor {
+  max-width: 980px;
+  border-top: 1px solid $su-color-black-20;
+  padding-top: 15px;
 
   .su-event-sponsor {
-    @include subheading;
+    @include type-e;
+  }
+
+  span {
+    font-weight: $su-font-semi-bold;
+  }
+
+  div,
+  span {
+    display: block;
+  }
+
+  @include grid-media-min('lg') {
+    span {
+      display: inline;
+    }
+
+    div {
+      display: inline;
+
+      &::after {
+        content: ", "
```

Comment:
If there are multiple available, why aren't the elements list elements instead of div and spans?

Reply:
Ah because it isn't a list on Events. I'll go look at that.

Reply:
The Events' Sponsor CT looks like this:
<img width="1496" height="452" alt="Screenshot 2025-07-22 at 12 09 12 PM" src="https://github.com/user-attachments/assets/1c808bd2-c445-448d-aaee-4373a9d905f9" />
The Opportunties Sponsor is a taxonomy.  What is the best approach here?